src/main/java/frc/robot/Constants.java

```diff
@@ -1,15 +1,50 @@
-// Copyright (c) FIRST and other WPILib contributors.
-// Open Source Software; you can modify and/or share it under the terms of
-// the WPILib BSD license file in the root directory of this project.
+/*----------------------------------------------------------------------------*/
```

Comment:
Please keep the updated copyright.

Reply:
What I meant was just use the latest copyright. Doesn't make sense to have two copyright headers.